R/pkg/R/DataFrame.R

```diff
@@ -1146,7 +1147,7 @@ setMethod("head",
 
 #' Return the first row of a SparkDataFrame
 #'
-#' @param x A SparkDataFrame
```

Comment:
I think similar to what you have for other functions, this could go to generic.R - do you have any other idea how to have functions working with multiple class documented?